src/main/java/xyz/luan/facade/HttpFacade.java

```diff
@@ -40,6 +45,29 @@
 
     private List<InputStream> customKeystoreCertificates = new ArrayList<>();
 
+    static {
+        try {
+            String[] methods = { "CONNECT", "TRACE", "PATCH" };
+
+            Field methodsField = HttpURLConnection.class.getDeclaredField("methods");
+
+            Field modifiersField = Field.class.getDeclaredField("modifiers");
+            modifiersField.setAccessible(true);
+            modifiersField.setInt(methodsField, methodsField.getModifiers() & ~Modifier.FINAL);
+
+            methodsField.setAccessible(true);
+
+            String[] oldMethods = (String[]) methodsField.get(null);
+            Set<String> methodsSet = new LinkedHashSet<>(Arrays.asList(oldMethods));
+            methodsSet.addAll(Arrays.asList(methods));
+            String[] newMethods = methodsSet.toArray(new String[0]);
+
+            methodsField.set(null, newMethods);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            System.out.println("Additional verbs could not be loaded: " + e);
```

Comment:
I think we can do System.err.println to be stronger about it (not as strong as crashing the app though).

Reply:
Done.